Gemfile

```diff
@@ -25,6 +25,9 @@ gem 'rails-controller-testing'
 gem 'mysql2', '0.3.20'
 
 #https://github.com/qertoip/transaction_retry
+# We don't use the isolation gem directly, but it needs to be
+# at the latest version to avoid errors
+# gem 'transaction_isolation', '1.0.5'
```

Comment:
Should we keep this if it's all commented out, and "latest version" will eventually drift? I'm fine with just the lock file change.

Reply:
The first two lines of the new comment might not be a bad idea, but I agree the third line is probably unnecessary.

Reply:
Okay, uncommented it so it's clearer.